cmd/dinghy.go

```diff
@@ -81,7 +84,7 @@ func dinghyRender(args []string) (string, error) {
 		TemplateRepo:    viper.GetString("modules"),
 		TemplateOrg:     "templateOrg",
 		Logger:          log.WithField("arm-cli-test", ""),
-		Client:          nil,
+		Client:          plank.New(plank.WithClient(httpClient), plank.WithFiatUser("")),
```

Comment:
We set fiat user to empty?

Reply:
I will change this line back to nil, I forgot to revert this change (was testing a pipelineID issue and forgot to clean this line).